### PR TITLE
ref(ts): Convert organizationGroupDetails Actions to typescript

### PR DESCRIPTION
**Pull request by priscilawebdev**
Convert organizationGroupDetails Actions to typescript

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationGroupDetails/actions/deleteAction.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+          <React.Fragment>
+            <Body>
+              {!hasFeature &&
+                typeof renderDisabled === 'function' &&
+                renderDisabled({...props, hasFeature, children: null})}
+              {t(
+                `Discarding this event will result in the deletion of most data associated with this issue and future events being discarded before reaching your stream. Are you sure you wish to continue?`
+              )}
+            </Body>
+            <div className="modal-footer">
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/actions/deleteAction.tsx:53]: Can we use Footer that's part of ModalRenderProps instead?

File: src/sentry/static/sentry/app/views/organizationGroupDetails/actions/deleteAction.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+  function openDiscardModal() {
+    openModal(renderDiscardModal);
+    analytics('feature.discard_group.modal_opened', {
+      org_id: parseInt(organization.id, 10),
+    });
+  }
+
+  return (
+    <div className="btn-group">
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/actions/deleteAction.tsx:78]: Is this replaceable with ButtonBar?

**Review by matejminar — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationGroupDetails/actions/deleteAction.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/actions/deleteAction.tsx:78]: I tried to do that, but there were some problems that I prefer to deal with in another PR

File: src/sentry/static/sentry/app/views/organizationGroupDetails/actions/deleteAction.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/actions/deleteAction.tsx:53]: I will check this, but I prefer to create a new PR with the updates 😉

**Pull request merged**